apps/gro/fields/email-address.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  'email-text': {
+    mixin: 'input-text',
```

Comment:
I thought it was.. yeah right you are. Didn't notice that - just renamed the file